ui/src/components/Organization/List.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames/bind';
+import style from 'src/components/Organization/style.module.scss';
+
+const c = classNames.bind(style);
+
+interface Organizations {
+  id: string;
+  name: string;
+  banner?: string;
+  square_logo?: string;
+}
+
+interface FieldProps {
+  list: Organizations[];
+}
+
+const List: React.FC<FieldProps> = ({ ...p }) => {
+  const organizations = p.list.map((e) => (
+    <div className={c('list-content')} key={e.id}>
+      <div className={c('list-section1')}>
+        <img className={c('item-wrapper')} src={e.square_logo} />
+        <span className={c('name-wrapper')}>{e.name}</span>
```

Comment:
Both these names (`item-wrapper` and `name-wrapper`) seem misplaced. These elements aren't wrapping anything, but are the atomic elements themselves. I recommend we name them `item` and `name` instead. 

A name should reflect the intention behind the value named. A better name for `<img` class would perhaps be `logo`.